src/app/static/src/app/components/verticalTabs.vue

```diff
@@ -49,7 +48,7 @@
         data() {
             return {
                 width: 0,
-                height: 0
+                height: 42
```

Comment:
realised we don't want this to be dynamic as it increases once the graph is present and really we just want it to always be the height of a nav-link which is 42px.